### DISCUSSION TITLE
Regional IPAdapter Workflow

### DISCUSSION THREAD

**Discussion by kakachiex2** [yolain/ComfyUI-Easy-Use#218] [Show and tell]
Hi YOLAIN, thanks for your hard work excellent node pack.
Ok to the point I'm trying to emulate "Invoke AI" regional layer guidance by color mask with IP-Adapter, your node pack have it but I don't know how to make it work correctly this is my workflow I will post the invoke AI video so you can have an idea of what I want to achieve.
- I want to render my line art with ControlNet and guide the style with IP-Adapter it work but the regional IP-Adapter dont see to work or my workflow is incorrect.
Invoke AI video: https://youtu.be/CLVylJAMIF8?si=V2hw35oOgn-XpBxX

My workflow:
![Regional IP-Adapter_WF_V1 0](https://github.com/yolain/ComfyUI-Easy-Use/assets/1762112/7933b9be-bc25-47fc-95d2-7f6ad866a1a8)

**Comment by yolain** [comment]
Thank you! I appreciate the compliment.
In addition to the pipe, ipadapter_params also need to connected.

For Example:
<img width="1621" alt="截屏2024-06-20 14 38 44" src="https://github.com/yolain/ComfyUI-Easy-Use/assets/73304135/b83f1214-2a71-4695-88fe-458bfc942eb3">

**Reply by kakachiex2** [reply]
Thanks @yolain

**Comment by kakachiex2** [comment]
Know is working perfectly thanks a lot:
![CNet LineArt Render-02](https://github.com/yolain/ComfyUI-Easy-Use/assets/1762112/b0641c8b-8916-494d-8134-2d5aac455ff5)
![Screenshot 2024-06-20 112627](https://github.com/yolain/ComfyUI-Easy-Use/assets/1762112/5a4bbb72-3e87-4bd3-8554-a8ffef53b172)

**Reply by yolain** [reply]
so cool, that's fantastic work！

**Comment by kakachiex2** [comment]
Hi @yolain 
Let me share this recent art I create thanks to your tools really easy for me to do this.
![Kaiyu8 composition](https://github.com/yolain/ComfyUI-Easy-Use/assets/1762112/69e7aed3-84f2-4ecd-9a88-28bba179e069)
